doc/cpp_recipe.rst

```diff
@@ -0,0 +1,230 @@
+Recipes
+===============
+
+An Arbor recipe is a description of a model without any information about how the
+model is to be executed. It  provides interface that the model building phase queries
+for information on cells, such as:
```

Comment:
Sentence starting line 5 is missing an article, but also convoluted phrasing. Rephrase more directly?

Reply:
Perhaps: _The recipe is queried during the model building phase to obtain cell information, such as:_